src/features/network-modifications/voltageLevel/section/CreateVoltageLevelSectionForm.tsx

```
@@ -0,0 +1,367 @@
+    if (!selectedOptionId || !data) {
+        return { position: -1, length: 0 };
+    }
+
+    for (const array of Object.values(data)) {
+        if (Array.isArray(array)) {
+            const position = array.indexOf(selectedOptionId);
+            if (position !== -1) {
+                return { position, length: array.length };
+            }
+        }
+    }
```

ESLint

[error] iterators/generators require regenerator-runtime, which is too heavyweight for this guide to allow them. Separately, loops should be avoided in favor of array iterations.

(no-restricted-syntax)